### ISSUE TITLE
Command failed with KeyError: !whois <@533620283748384779>

### ISSUE THREAD

**Issue by vorpal-buildbot** [PennyDreadfulMTG/Penny-Dreadful-Tools#6591]
```
Traceback (most recent call last):
  File "/home/discord/Penny-Dreadful-Discord-Bot/discordbot/command.py", line 73, in handle_command
    await method(Commands, client=client, channel=message.channel, args=args, author=message.author)
  File "/home/discord/Penny-Dreadful-Discord-Bot/discordbot/command.py", line 806, in whois
    await send(channel, f"{mention.group(0)} is **{person['name']}** on MTGO")
KeyError: 'name'

```

Reported on Discord by bakert#2193

**Comment by bakert** [comment]
This is from saying `!whois @tp0nt3`

**Comment by bakert** [comment]
Theoretic fix is in 60aad498. Will check before closing.
